include/klee/util/Bits.h

```diff
@@ -88,9 +95,11 @@ namespace klee {
     // @pre((x&(x-1)) == 0)
     // @post((1 << retval) == x)
     inline unsigned indexOfSingleBit(uint64_t x) {
+      assert((x & (x - 1)) == 0);
       unsigned res = bits32::indexOfSingleBit((unsigned) (x | (x>>32)));
-      if (x&((uint64_t) 0xFFFFFFFF << 32))
-	  res += 32;
+      if (x & ((uint64_t)0xFFFFFFFF << 32))
```

Comment:
Dan, I'm not sure I follow your argument here.  Who forces you "into throwing as many changes into a PR as possible even though they might not be related"?!  We are very slow to review PRs and we discussed the reasons why before, but the point is certainly not to make it artificially hard to contribute.